arapp.json

```diff
@@ -14,7 +14,7 @@
   "environments": {
     "default": {
       "network": "rpc",
-      "appName": "placeholder-app-name.open.aragonpm.eth"
+      "appName": "counter.aragonpm.eth"
```

Comment:
I think we want to keep the placeholder; `create-aragon-app` searches and replaces it